StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/Models/FinancialConnectionsTheme.swift

```diff
@@ -58,3 +58,16 @@ extension FinancialConnectionsTheme {
         }
     }
 }
+
+extension FinancialConnectionsTheme? {
+    var spinnerColor: UIColor {
+        switch self {
+        case .linkLight:
+            return .linkGreen200
+        case .light:
+            return .brand500
+        case .none:
+            return .neutral200
+        }
+    }
+}
```

Comment:
I initially thought it be easier to just do something like:

`spinner.backgroundColor = theme?.spinnerColor ?? .neutral200` (or pass in the spinner color as the configuration)

but I guess sometimes you might need the whole theme instead of one color? another is to introduce a new theme "neutral" or something...but that seems a bit over engineery too

then we likely don't want to ONLY be able to access a color IF the theme is non-null; so maybe a better way is to write it something like 

```
extension FinancialConnectionsTheme? {
    var spinnerColor: UIColor {
        switch self {
        case .all: // this is psuedocode that doesn't compile
            return theme.spinnerColor
        case .none:
            return .neutral200
        }
    }
    ```

Reply:
Good points! The reason for putting the `spinnerColor` extension variable on the optional theme was to keep all the color decisions centralized in one place. Though, I agree with the point of only being able to access the `spinnerColor` when the theme is optional, so I'll update this with your suggestion!